jupyterhub/app.py

```diff
@@ -2032,10 +2043,13 @@ async def init_role_creation(self):
             old_role = orm.Role.find(self.db, name=role_name)
             if old_role:
                 if not set(role_spec['scopes']).issubset(old_role.scopes):
-                    app_log.warning(
+                    self.log.warning(
```

Comment:
These don't change anything, but good to be consistent referencing `self.log`. They point to the same logger, ultimately.